src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2161,6 +2161,14 @@ internal static CSharpSyntaxNode GetStandaloneNode(CSharpSyntaxNode node)
                         return parent;
                     }
 
+                    break;
+
+                case SyntaxKind.NameColon:
```

Comment:
>NameColon [](start = 32, length = 9)

I didn't understand this. A subpattern is not an expression, but this method should return "the containing expression that is actually a language expression".

Reply:
Will adjust comment. It is not just for expressions, it is for anything that can be used for GetSymbolInfo. That now includes a subpattern.

---
In reply to: [191021322](https://github.com/dotnet/roslyn/pull/27160#discussion_r191021322) [](ancestors = 191021322)